### ISSUE TITLE
Charges overview not working

### ISSUE THREAD

**Issue by BDThug** [adriankumpf/teslamate#241]
In my Grafana overview, the charges overview is not working. The error is: `pq: column "charge_energy_used" does not exist`

I installed Teslamate on my raspberry pi (Raspian) with the provided docker compose yml file.  

I am on Teslamate 1.11 (why not the latest version with this docker compose file?)

I also see my Tesla is not sleeping at all since I installed Teslamate. 

What can I do to fix the Charges view? 
And also update to the latest version and I want my Tesla to sleep again :)

**Comment by adriankumpf** [comment]
You seem to be using the outdated Teslamate Docker image (v1.11), whereas your Grafana image is up to date. Try `docker-compose pull`.

**Comment by BDThug** [comment]
> You seem to be using the outdated Teslamate Docker image (v1.11), whereas your Grafana image is up to date. Try `docker-compose pull`.
> 
> Regarding the sleep issue there is this help page: https://github.com/adriankumpf/teslamate/blob/master/docs/configuration/sleep.md

Thank you. I will try this tonight. Regarding the Sleep issue I already looked into that. 
There is no data in Vampire Drain now, my battery is down from 76 to 74 in 6 hours, nothing reported. Car state is online the whole time. (or charging, driving)

My settings are: 
![image](https://user-images.githubusercontent.com/54153869/68307716-fcdd7180-00ab-11ea-8af4-6734870836ae.png)


At first my settings where the default of 21 and 15 minutes. 
I did not open my Tesla app(closed completely) on my phone and the phone is not connected to the car via Bluetooth.
